backend/infrahub/core/branch/tasks.py

```diff
@@ -156,14 +167,24 @@ async def rebase_branch(branch: str, context: InfrahubContext, service: Infrahub
     # -------------------------------------------------------------
     # Generate an event to indicate that a branch has been rebased
     # -------------------------------------------------------------
-    # TODO Add account information
-    await service.event.send(
-        event=BranchRebasedEvent(
-            branch_name=obj.name,
-            branch_id=str(obj.uuid),
-            meta=EventMeta.from_context(context=context, branch=registry.get_global_branch()),
-        )
+    rebase_event = BranchRebasedEvent(
+        branch_name=obj.name, branch_id=str(obj.uuid), meta=EventMeta(branch=obj, context=context)
     )
+    events: list[InfrahubEvent] = [rebase_event]
+    changelog_collector = DiffChangelogCollector(diff=default_branch_diff, branch=obj, db=db, migrations=migrations)
```

Comment:
`migrations` seems never filled?

Reply:
There's a section within an if statement:

```python
                # Execute the migrations
                migrations = await merger.calculate_migrations(target_schema=updated_schema)
```

The top one is just to avoid having an unbound variable if we don't hit this part of the code.